governance/mixed.go

```diff
@@ -0,0 +1,188 @@
+package governance
+
+import (
+	"github.com/klaytn/klaytn/blockchain/types"
+	"github.com/klaytn/klaytn/common"
+	"github.com/klaytn/klaytn/consensus/istanbul"
+	"github.com/klaytn/klaytn/params"
+	"github.com/klaytn/klaytn/storage/database"
+)
+
+type MixedEngine struct {
+	// Subordinate engines
+	defaultGov *Governance
```

Comment:
Not the `Engine` interface?

Reply:
Maybe I should use `HeaderEngine` instead.